dist/container.js

```diff
@@ -2517,40 +2542,49 @@
 		{
 			set: function(paused)
 			{
-				this._paused = paused;
-
-				if (this.client)
+				if (this._disablePause)
```

Comment:
this line of code needs a exclamation mark in the front of it as the disable functionality is working backwards